cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -96,7 +98,7 @@ export class PassportAuth implements Auth {
    */
   public authenticate(defaultRedirect: string) {
     return passport.authenticate('local', {
-      failureRedirect: this.loginRoute,
+      failureRedirect: this.loginErrorRoute,
```

Comment:
We can/should pass this here. As every authenticate may have different error route.